.changeset/wild-owls-doubt.md

```diff
@@ -0,0 +1,16 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+The built-in module list has been trimmed down when using the new Backend System. Provider specific modules should now be installed with `backend.add` to provide additional actions to the scaffolder. These modules are as follows:
```

Comment:
Let's update https://backstage.io/docs/backend-system/building-backends/migrating#the-scaffolder-plugin as well